source-s3/main.go

```diff
@@ -255,7 +255,7 @@ func main() {
 			"region": {
 				"type":        "string",
 				"title":       "AWS Region",
-				"description": "The name of the AWS region where the S3 stream is located",
+				"description": "The name of the AWS region where the S3 stream is located. \"us-east-1\" is a popular default you can try, if you're unsure what to put here.",
```

Comment:
Is "S3 stream" the way _we're_ choosing to talk about it here? Or would this make more sense to the user if we just said "S3 bucket"?

Reply:
lol I didn't even notice that. I'm assuming it's an artifact of copypasta from source-kinesis. Changed it to "bucket".